### DISCUSSION TITLE
create service with manual start only

### DISCUSSION THREAD

**Discussion by kenzanin** [finit-project/finit#178] [Q&A] [answered]
Dear finit users,

how to create service with manual start only without using cond usr/<some cond>, i want to use initctl start service rather than initctl cond set some_condition_in_service.

thanks.

**Comment by troglobit** [comment, marked as answer]
Should be sufficient to add the `manual:yes` modifier, e.g.:

    service manual:yes /sbin/foo -- Foo daemon
